autopayroll-H2Database/workflows/src/main/kotlin/com/autopayroll/flows/RequestFlow.kt

```diff
@@ -0,0 +1,80 @@
+package com.autopayroll.flows
+
+import co.paralleluniverse.fibers.Suspendable
+import com.autopayroll.contracts.PaymentRequestContract
+import com.autopayroll.states.PaymentRequestState
+import net.corda.core.contracts.requireThat
+import net.corda.core.flows.*
+import net.corda.core.utilities.ProgressTracker
+import net.corda.core.transactions.TransactionBuilder
+import net.corda.core.flows.FlowSession
+import net.corda.core.transactions.SignedTransaction
+import net.corda.core.flows.FinalityFlow
+import net.corda.core.flows.CollectSignaturesFlow
+import net.corda.core.identity.CordaX500Name
+import net.corda.core.identity.Party
+
+
+// *********
+// * Flows *
+// *********
+@InitiatingFlow
+@StartableByRPC
+class RequestFlowInitiator(
+        private val amount:String,
+        private val towhom:Party
+) : FlowLogic<SignedTransaction>() {
+    companion object {
+        object GENERATING_TRANSACTION : ProgressTracker.Step("Generating transaction for between accounts")
+        object PROCESS_TRANSACTION : ProgressTracker.Step("PROCESS transaction with our private key.")
+        object FINALISING_TRANSACTION : ProgressTracker.Step("Obtaining notary signature and recording transaction.")
+
+        fun tracker() = ProgressTracker(
+                GENERATING_TRANSACTION,
+                PROCESS_TRANSACTION,
+                FINALISING_TRANSACTION
+        )
+    }
+
+    override val progressTracker = tracker()
+
+    @Suspendable
+    override fun call(): SignedTransaction{
+        // Initiator flow logic goes here
+
+
```

Comment:
You seem to have copied this white space and comment from somewhere as it keeps coming up.